src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -26,20 +35,35 @@ namespace orbit_object_utils {
 
 namespace {
 
+[[nodiscard]] uint64_t ComputeRelativeAddress(
+    uint64_t offset_in_section, uint16_t section, uint64_t load_bias,
+    const llvm::FixedStreamArray<llvm::object::coff_section>& section_headers) {
+  // The address in PDB files is a relative virtual address (RVA), to make the address
+  // compatible with how we do the computation, we need to add both the load bias (ImageBase for
+  // COFF) and the virtual offset of the section. Note: The segments are numbered starting at 1 and
```

Comment:
I don't think we should mention load bias in this function: this is PE/COFF-related code, so we can just talk about ImageBase.
And now that the term "load bias" is no longer involved, we shouldn't say "to make the address compatible with how we do the computation": we simply want to compute the Virtual Address.